deployments/opbnbtestnet_addresses.json

```diff
@@ -10,6 +10,7 @@
     "ComptrollerImpl": "0x0012875a7395a293Adfc9b5cDC2Cfa352C4cDcD3",
     "Comptroller_Core": "0x2FCABb31E57F010D623D8d68e1E18Aed11d5A388",
     "DefaultProxyAdmin": "0xB1281ADC816fba7df64B798D7A0BC4bd2a6d42f4",
+    "JumpRateModelV2_base0bps_slope1000bps_jump8000bps_kink8000bps_bpy63072000": "0x28EDBbeF282e1B500Ae82D0a1A01A2093157D8c7",
```

Comment:
verify the address

Reply:
It will be verified after confirming with Nodereal the way to do it